model_compression_toolkit/target_platform_capabilities/target_platform/targetplatform2framework/attach2keras.py

```diff
@@ -0,0 +1,106 @@
+
+if FOUND_SONY_CUSTOM_LAYERS:
+    from sony_custom_layers.keras.object_detection.ssd_post_process import SSDPostProcess
+
+if version.parse(tf.__version__) >= version.parse("2.13"):
+    from keras.src.layers import Conv2D, DepthwiseConv2D, Dense, Reshape, ZeroPadding2D, Dropout, \
+        MaxPooling2D, Activation, ReLU, Add, Subtract, Multiply, PReLU, Flatten, Cropping2D, LeakyReLU, Permute, \
+        Conv2DTranspose, Identity, Concatenate, BatchNormalization, Minimum, Maximum
+else:
+    from keras.layers import Conv2D, DepthwiseConv2D, Dense, Reshape, ZeroPadding2D, Dropout, \
+        MaxPooling2D, Activation, ReLU, Add, Subtract, Multiply, PReLU, Flatten, Cropping2D, LeakyReLU, Permute, \
+        Conv2DTranspose, Concatenate, BatchNormalization, Minimum, Maximum
+
+from model_compression_toolkit import DefaultDict
+from model_compression_toolkit.target_platform_capabilities.constants import KERNEL_ATTR, BIAS, \
+    BIAS_ATTR, KERAS_KERNEL, KERAS_DEPTHWISE_KERNEL
+from model_compression_toolkit.target_platform_capabilities.schema.mct_current_schema import OperatorSetNames
+from model_compression_toolkit.target_platform_capabilities.target_platform import LayerFilterParams
+from model_compression_toolkit.target_platform_capabilities.target_platform.targetplatform2framework.attach2fw import \
+    TpcAttach2Fw
+
+
+class TpcAttach2Keras(TpcAttach2Fw):
+    def __init__(self):
+        super().__init__()
+
+        self._opset2layer = {
+            OperatorSetNames.OPSET_CONV.value: [Conv2D, tf.nn.conv2d],
+            OperatorSetNames.OPSET_DEPTHWISE_CONV.value: [DepthwiseConv2D, tf.nn.depthwise_conv2d],
+            OperatorSetNames.OPSET_CONV_TRANSPOSE.value: [Conv2DTranspose, tf.nn.conv2d_transpose],
+            OperatorSetNames.OPSET_FULLY_CONNECTED.value: [Dense],
+            OperatorSetNames.OPSET_CONCATENATE.value: [tf.concat, Concatenate],
+            OperatorSetNames.OPSET_STACK.value: [tf.stack],
+            OperatorSetNames.OPSET_UNSTACK.value: [tf.unstack],
+            OperatorSetNames.OPSET_GATHER.value: [tf.gather, tf.compat.v1.gather],
+            OperatorSetNames.OPSET_EXPAND.value: [],
+            OperatorSetNames.OPSET_BATCH_NORM.value: [BatchNormalization],
+            OperatorSetNames.OPSET_RELU.value: [tf.nn.relu, ReLU],
+            OperatorSetNames.OPSET_RELU6.value: [tf.nn.relu6],
+            OperatorSetNames.OPSET_LEAKY_RELU.value: [tf.nn.leaky_relu, LeakyReLU],
+            OperatorSetNames.OPSET_HARD_TANH.value: [LayerFilterParams(Activation, activation="hard_tanh")],
+            OperatorSetNames.OPSET_ADD.value: [tf.add, Add],
+            OperatorSetNames.OPSET_SUB.value: [tf.subtract, Subtract],
+            OperatorSetNames.OPSET_MUL.value: [tf.math.multiply, Multiply],
+            OperatorSetNames.OPSET_DIV.value: [tf.math.divide, tf.math.truediv],
+            OperatorSetNames.OPSET_MIN_MAX.value: [tf.math.minimum, tf.math.maximum, Minimum, Maximum],
```

Comment:
Split two operator  min & max

Reply:
@lior-dikstein this is more relevant to your PR